packages/editor/src/components/block-switcher/index.js

```diff
@@ -58,7 +58,20 @@ export class BlockSwitcher extends Component {
 		const hasStyles = blocks.length === 1 && get( blockType, [ 'styles' ], [] ).length !== 0;
 
 		if ( ! hasStyles && ! possibleBlockTransformations.length ) {
-			return null;
+			if ( blocks.length > 1 ) {
```

Comment:
Why do we care about whether there's a multi-selection here, when in the rendering of the `IconButton` below, we use `blockType.icon` (the first selected block's icon) whether or not it's a multi-selection.

Arrived here via rebasing of #12828, where this code is problematic and difficult to resolve due to the inconsistent forking logic.